### ISSUE TITLE
Not working correctly?

### ISSUE THREAD

**Issue by BlueHorse259** [mschae23/grind-enchantments#73]
hello, 

I'm playing on version 1.21.4  as far as I know the mod should be compatible with this version but for some reason I can not put a book into the grinder 

I can put helmets and swords and all into it but no book help?

**Comment by mschae23** [comment]
Yes, if you have version `4.0.0+1.21.4-rc3`, the mod should be compatible. Is this happening in singleplayer? Multiplayer? If it's the latter, make sure the mod is installed on the server.
